json_tricks/decoders.py

```diff
@@ -117,6 +117,23 @@ def noenum_hook(dct):
 	return dct
 
 
+def pathlib_hook(dct):
+		if not isinstance(dct, dict):
```

Comment:
This seems to be indented with double tabs, while the rest of the code is single tabs

Reply:
Sorry! I didn't realize the file was formatted with tabs when I first started editing, so I had to go back and convert everything from spaces to tabs. I'll fix this.